javascript/packages/orchestrator/src/chain-spec.ts

```diff
@@ -580,6 +580,57 @@ export async function getChainIdFromSpec(specPath: string): Promise<string> {
   });
 }
 
+export async function customizePlainRelayChain(
+  specPath: string,
+  networkSpec: ComputedNetwork,
+): Promise<void> {
+  // Relay-chain spec customization logic
```

Comment:
since we are refactoring things - should we add also a try/catch in these functions?

Reply:
Sounds good, I will add now.

Reply:
done, thanks!